be/src/vec/aggregate_functions/aggregate_function_window.h

```diff
@@ -210,26 +210,30 @@ class WindowFunctionNTile final
 struct Value {
 public:
     bool is_null() const { return _is_null; }
-    StringRef get_value() const { return _value; }
-
     void set_null(bool is_null) { _is_null = is_null; }
-    void set_value(StringRef value) { _value = value; }
+    StringRef get_value() const { return _ptr->get_data_at(_offset); }
+
+    void set_value(StringRef ref) {
```

Comment:
Whether use `StringRef` to represent column_ptr and offset is too tricky?

Reply:
just keep same with `copy_value` do make sure the interface is same